### PR TITLE
Add livewire component discovery solution.

### PR DESCRIPTION
**Pull request by ZeProf2Code**

### CODE REVIEW

File: src/SolutionProviders/MissingLivewireComponentSolutionProvider.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+class MissingLivewireComponentSolutionProvider implements HasSolutionsForThrowable
+{
+    public function canSolve(Throwable $throwable): bool
+    {
+        if (! class_exists(ComponentNotFoundException::class)) {
+            return false;
+        }
+        if (! $throwable instanceof \Livewire\Exceptions\ComponentNotFoundException) {
+            return false;
+        }
```
Comment (freekmurze) [inline src/SolutionProviders/MissingLivewireComponentSolutionProvider.php:20]: Also check here if the class `Livewire\LivewireComponentsFinder` exists.

File: src/SolutionProviders/MissingLivewireComponentSolutionProvider.php [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+use Throwable;
+
+class MissingLivewireComponentSolutionProvider implements HasSolutionsForThrowable
+{
+    public function canSolve(Throwable $throwable): bool
+    {
+        if (! class_exists(ComponentNotFoundException::class)) {
+            return false;
+        }
+        if (! $throwable instanceof \Livewire\Exceptions\ComponentNotFoundException) {
```
Comment (freekmurze) [inline src/SolutionProviders/MissingLivewireComponentSolutionProvider.php:18]: Import this class

**Review by freekmurze — CHANGES_REQUESTED** [review]
Looks good, just a few nitpicks. Could you add a screenshot in the comments of how this solution looks like? Thanks!

**Comment by ZeProf2Code** [comment]
image
How it looks ^

**Comment by freekmurze** [comment]
Thank you!

**Pull request merged**